doc/api/http.md

```diff
@@ -3632,6 +3635,11 @@ changes:
   * `rejectNonStandardBodyWrites` {boolean} If set to `true`, an error is thrown
     when writing to an HTTP response which does not have a body.
     **Default:** `false`.
+  * `fastDump` {boolean} If set to `true`, request body for `HEAD` and `GET`
```

Comment:
I'm not a big fan of `fastDump` as a name for this option. Perhaps, `tryFastClose`, or something similar, that is more descriptive of what is happening. `dump` is just too generic.